README.md

```diff
@@ -80,6 +80,8 @@ If the filename ends with `.url` suffix, the content will be processed as a URL
 | `SCRIPT`                   | Absolute path to shell script to execute after a configmap got reloaded. It runs before calls to `REQ_URI`                                                                                                                                                                                                                          | false    | -                                         | string  |
 | `ERROR_THROTTLE_SLEEP`     | How many seconds to wait before watching resources again when an error occurs                                                                                                                                                                                                                                                       | false    | `5`                                       | integer |
 | `SKIP_TLS_VERIFY`          | Set to `true` to skip tls verification for kube api calls                                                                                                                                                                                                                                                                           | false    | -                                         | boolean |
+| `SKIP_GRAFANA_TLS_VERIFY`          | Set to `true` to skip tls verification for Grafana api calls (for reloading certain resources).
```

Comment:
Please don't mention Grafana here. I understand the sidecar is especially popular for usage with grafana, but it is certainly not limited to it. This setting affects all http requests made based on `REQ_URL` config and for `*.url` triggered requests. So I suggest naming it `REQ_SKIP_TLS_VERIFY` then and document the affected calls (similar to `REQ_PASSWORD` I think).